src/Umbraco.Web/PropertyEditors/RichTextPropertyEditor.cs

```diff
@@ -63,14 +75,16 @@ internal class RichTextPropertyValueEditor : DataValueEditor, IDataValueReferenc
             private readonly HtmlImageSourceParser _imageSourceParser;
             private readonly HtmlLocalLinkParser _localLinkParser;
             private readonly RichTextEditorPastedImages _pastedImages;
+            private readonly IImageUrlGenerator _imageUrlGenerator;
 
-            public RichTextPropertyValueEditor(DataEditorAttribute attribute, IUmbracoContextAccessor umbracoContextAccessor, HtmlImageSourceParser imageSourceParser, HtmlLocalLinkParser localLinkParser, RichTextEditorPastedImages pastedImages)
+            public RichTextPropertyValueEditor(DataEditorAttribute attribute, IUmbracoContextAccessor umbracoContextAccessor, HtmlImageSourceParser imageSourceParser, HtmlLocalLinkParser localLinkParser, RichTextEditorPastedImages pastedImages, IImageUrlGenerator imageUrlGenerator)
```

Comment:
We need to have the old constructor with ` [Obsolete("Use the constructor which takes an IImageUrlGenerator")]`

Reply:
The reason I didn't keep the old constructor was because this was an Internal class, so it can't be referenced outside the solution.  Since we updated all the references in the solution, there is no reason to keep the old constructor.